.gitignore

```diff
@@ -7,6 +7,7 @@ node_modules
 
 # build
 dist
+types
```

Comment:
As I noted I pulled these up as an example but feel free to use it in `dist/types` if you liked that config.

Reply:
I do prefer them to be generated at `dist/types`.